src/utils/polling.js

```diff
@@ -61,20 +97,31 @@ async function pollSubscriptionTopics() {
       auth: { bearer: accessToken },
     };
 
+    const resourceToPoll = namedEventToResourceType(topic);
+    const mostRecentPoll = db.select('polling', (p) => p.resource === resourceToPoll);
+    const lastUpdated = mostRecentPoll.length === 0 ? null : mostRecentPoll[0].timestamp;
     const fhirClient = mkFhir(options);
     fhirClient
-      .search(getSearchQuery(topic))
+      .search(getSearchQuery(resourceToPoll, lastUpdated))
       .then((response) => {
         const { data } = response;
 
+        // Add poll to DB
+        const poll = {
+          id: uuidv4(),
+          timestamp: new Date(Date.now()).toISOString(),
```

Comment:
The `Date.now()` isn't needed here
```suggestion
          timestamp: new Date().toISOString(),
```